package.json

```diff
@@ -43,7 +43,7 @@
     "@types/supertest": "^6.0.0",
     "@typescript-eslint/eslint-plugin": "^8.0.0",
     "@typescript-eslint/parser": "^8.0.0",
-    "eslint": "^9.0.0",
+    "eslint": "^8.57.1",
```

Comment:
why ?

Reply:
it was either downgrade the eslint or re write the eslint file, seemed simpler to downgrade.